main.py

```diff
@@ -228,15 +230,6 @@ def to_finite(self, mat):
         return final_finite
 
 
-# Utility function adding a value to a queue style iterable, maintaining sorting from smallest to largest
-def min_add(iterable, val):
-    for i, item in enumerate(iterable):
-        if val < item:
-            iterable = iterable[:i] + [val] + iterable[i:(len(iterable) - 1)]
-            return iterable, True
-    return iterable, False
-
-
 class Material:
     def __init__(self, name, E, G, Iy, Iz, J, A):
         self.name = name
```

Comment:
are these optional args? I'd make them `E=None, G=None,` etc to denote optional args
this should fail lint checking